File: src/main/Java/boj_step6/boj2941.java
import java.io.*;

public class boj2941 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        String[] croa = { "c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z=" };
        String line = reader.readLine();
        for (String key: croa) {
            line = line.replace(key, "A");
        }
        writer.append(String.valueOf(line.length()));
        writer.close();
    }
}

File: src/main/Kotlin/others/boj2941.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val croatia = arrayOf("c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z=")
    var str = r.readLine()!!
    croatia.forEach {
        str = str.replace(it, "A")
    }
    w.write(str.length.toString())
    w.close()
}